api_verifier/cloudbuild.yaml

```diff
@@ -0,0 +1,17 @@
+# Config for building with Google Container Builder
+#
+# This produces a container with two tags, "latest" and _RC_NAME, which must be
+# specified via a command-line flag.
+#
+# Run with:
+#   gcloud container builds submit --config cloudbuild.yaml . \
+#     --substitutions=_RC_NAME=20180101-RC00
```

Comment:
Missing parameter for PROJECT_ID

Reply:
PROJECT_ID is special-cased by container builder, so you don't need to provide it: https://cloud.google.com/container-builder/docs/configuring-builds/substitute-variable-values

Reply:
Good to know.